metadata/groups.yaml

```diff
@@ -68,15 +68,23 @@
   shorthand: WB
 -
   label: 'IntAct'
-  id: https://www.ebi.ac.uk/intact
+  id: http://www.ebi.ac.uk/intact
   shorthand: IntAct
 -
-  label: 'FlyBase'
+  label: 'SPOT (EBI)'
+  id: http://www.ebi.ac.uk/spot
+  shorthand: 'SPOT'
+-
+  label: EMBL-EBI
+  id: http://www.ebi.ac.uk
+  shorthand: EBI
+-
+  label: 'FB'
```

Comment:
Why this distinction? `FlyBase` would be the more readable version...

Reply:
Chris said that the `FlyBase` needed to be the shorthand, as that matches the needed field in GAF. But many of the users have `FB` as their group. So I wanted `FB` to be the label in order to detect that when a user says `FB` they mean `FlyBase`.

Reply:
Sorry, I'm not understanding why not just having the label be "FlyBase" too?

Reply:
Because people had created their user organization as `FB`. I would have had to change people's organization from `FB` to `FlyBase` when making the repair script. But if a user's org should not be `FB` we could change that their in users.yaml.